setup.py

```diff
@@ -3,95 +3,92 @@
 from os import path
 
 from jupyter_packaging import (
-    create_cmdclass, install_npm, ensure_targets,
-    combine_commands, get_version
+    create_cmdclass,
+    install_npm,
+    ensure_targets,
+    combine_commands,
+    get_version,
 )
 
 pjoin = path.join
 
-name = 'nbcelltests'
+name = "nbcelltests"
 here = path.abspath(path.dirname(__file__))
-jshere = path.join(here, 'js')
-version = get_version(pjoin(here, name, '_version.py'))
+jshere = path.join(here, "js")
+version = get_version(pjoin(here, name, "_version.py"))
 
-with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+with open(path.join(here, "README.md"), encoding="utf-8") as f:
     long_description = f.read().replace("\r\n", "\n")
 
 requires = [
     'backports.tempfile ; python_version < "3"',
-    'flake8',
-    'nbconvert',
-    'nbformat>=4.0.0',
-    'nbval>=0.9.1',
-    'notebook',
-    'parameterized',
-    'pytest>=4.4.0',
-    'pytest-cov',
-    'pytest-html>=1.20.0',
+    "flake8",
+    "nbconvert",
+    "nbformat>=4.0.0",
+    "nbval>=0.9.1",
+    "notebook",
+    "parameterized",
+    "pytest>=4.4.0",
+    "pytest-cov",
+    "pytest-html>=1.20.0",
 ]
 
 dev_requires = requires + [
-    'autopep8',
-    'beautifulsoup4',
-    'bump2version',
-    'jupyterlab>=3.0.0 ; python_version > "3"',
-    'jupyterlab ; python_version < "3"',
```

Comment:
Not clear to me how these changes relate to moving to black. So, I am checking with you that they are intentional. 

If they are intentional (e.g. because someone working on nbcelltests does not necessarily need to install lab - which I think is true), then they're fine with me.